social-net/frontend/src/components/Item.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+class Item extends React.Component {
+    link = '?c=' + this.props.item[1];
```

Comment:
You should do it in constructor or in method of life cycle like "componentWillMount"